core/src/main/scalajvm/sttp/tapir/Defaults.scala

```diff
@@ -1,7 +1,15 @@
 package sttp.tapir
 
+import sttp.tapir.model.SttpFile
+
 import java.io.File
+import scala.concurrent.Future
 
 object Defaults {
   def createTempFile: () => File = () => File.createTempFile("tapir", "tmp")
+
+  def deleteFiles(): Seq[SttpFile] => Future[Unit] = { files =>
```

Comment:
maybe the default symmetrically should be `deleteFile: SttpFile => Unit`? Then interpreters could integrate with that. Also it will be easier for the user to provide a function to delete a single file, we can handle the sequencing in the interpreter